app/services/activity_creator.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class ActivityCreator
```

Comment:
I really see no reason to use a class for this. This should me a module.

Reply:
ok